src/ow1/heroes/dva.opy

```diff
@@ -24,4 +24,12 @@ rule "[dva.opy]: Set default missile cooldown":
     @Condition eventPlayer.isUsingAbility2()
     
     waitUntil(not eventPlayer.isUsingAbility2(), 9999)
-    eventPlayer.setAbilityCooldown(Button.ABILITY_2, OW1_DVA_MISSILE_COOLDOWN)
\ No newline at end of file
+    eventPlayer.setAbilityCooldown(Button.ABILITY_2, OW1_DVA_MISSILE_COOLDOWN)
+
+rule "[dva.opy]: Make melee cancel DVa boosters":
+    @Event eachPlayer
+    @Hero dva
+    @Condition eventPlayer.isMeleeing() == true
+    @Condition eventPlayer.isUsingAbility1() == true
+    
+    eventPlayer.forceButtonPress(Button.ABILITY_1)
```

Comment:
The animation is a little off because cancelling booster during meleeing animation cancels the melee animation.

The correct behavior should be to cancel booster right before starting the melee animation.